docs/sources/administration/service-accounts/about-service-accounts.md

```diff
@@ -0,0 +1,46 @@
+---
+title: About service accounts
+menuTitle: About service accounts
+aliases: [docs/sources/manage-service-accounts/_index.md, 
+docs/sources/administration/service-accounts/about-service-accounts.md,
+docs/sources/administration/service-accounts/create-service-accounts.md]
+description: "This contains detailed information about service accounts in Grafana"
+weight: 30
+---
+
+# About service accounts in Grafana
+
+A service account is a regular user account used to run automated or compute workloads which are intended for programmatic use either internally for a Grafana feature or to communicate outside of the Grafana environment. Applications use service account tokens to authorize themselves as a service account.
+
+A common use case for creating a service account is to perform operations on automated or triggered tasks. You can use service accounts to:
+
+- Schedule reports for specific dashboards to be delivered on a daily/weekly/monthly basis
+- Define alerts in your system to be used in Grafana
+- Set up to manage users and permissions across the organization to an external auth provider
+- Machine to machine communication
+
+It can also be used in combination with FGAC to grant specific scopes.
+
+A service account can be associated with multiple api keys. As a result, we recommend starting off by creating one service account per use case.
+
+> **Note:** Service accounts act on an organization level, if you have the same task that is needed for multiple organizations, we recommend provisioning service accounts for each organization.
+
+---
+
+## Grafana Service account vs API key
+
+We have created service accounts for improved access control of programmatic access to Grafana. A service account token is a token that is used to authenticate a service account to Grafana and is very similar to an API key.
+
+We recommend to start using service accounts over API keys.
+
+### Service accounts benefits over API keys
```

Comment:
I would actually stop talking about API keys moving forward, and just go right to the benefits of service accounts.